app/forms/candidate_interface/continuous_applications/find_course_selection_step.rb

```diff
@@ -39,6 +49,10 @@ def study_mode
       def course
         Course.find(course_id)
       end
+
+      def confirm_answer
+        ActiveModel::Type::Boolean.new.cast(confirm)
```

Comment:
You're calling `.blank?` and `.present?` on this method. Worth doing this maybe?

```ruby
def confirm_answer?
  ActiveModel::Type::Boolean.new.cast(confirm).present?
end
```

and then use `confirm_answer?` and `!confirm_answer?`